src/renderer/vt/paint.cpp

```diff
@@ -538,15 +547,15 @@ using namespace Microsoft::Console::Types;
             RETURN_IF_FAILED(_EraseLine());
         }
     }
-    else if (_newBottomLine)
+    else if (_newBottomLine && coord.Y == _lastViewport.BottomInclusive())

```

Comment:
oh it was the same issue as before! should we pull this out into a local, `dirtyingNewBottomLine`? That'll make sure we don't use _newBottomLine as much in the body here..

Reply:
Eh probably